### ISSUE TITLE
Feature: Account Abstraction - Custom Validation Signature Scheme

### ISSUE THREAD

**Issue by julio4** [NethermindEth/StarknetByExample#191]
### Description

Account Abstraction replace Externally Owned Accounts (EOA). Accounts are implemented as smart contract called smart wallet. 

Account Abstraction allows for validation abstraction, which means that it is possible to implement the validation part of transaction inside a smart wallet in any way possibles. 

In this example, we want to show how we can implement a custom validation method using a different signature scheme.

### Criteria:

- Clearly explain the separation between the signer and the account.
- Focus on the implementation. There’s already lots of explanations about it in the Starknet Book, give a succinct overview of the most important points.
- SNIP-6 will be explained in #188 (by @Jemiiah), so focus only on the custom validation part.

### Resources:

- [Starknet Book - Account Abstraction](https://book.starknet.io/ch04-00-account-abstraction.html)

### ODHack

To be eligible for additional rewards, be sure to review and follow the [ODHack Common Guidelines](https://www.notion.so/ODHack-Common-Guidelines-b9c6b6a4ac4146d087185568aca38a3f?pvs=21) and [Contributing Guidelines](https://github.com/NethermindEth/StarknetByExample/blob/main/CONTRIBUTING.md#contributing).
Be sure to join the telegram group and introduce yourself.

**Comment by KodeSage** [comment]
Hi @julio4 
my name is James Harrison, I will love to work on this tasks.

**Comment by julio4** [comment]
@KodeSage I'll assign you. Please open a draft PR as soon as possible for early feedback on the specific validation scheme chosen. Please do not use generative AI to write explanations as well.

**Comment by Jemiiah** [comment]
@julio4 i would love to work on this incase KodeSage is not interested. i know what to do go about the work

**Comment by KodeSage** [comment]
I'm still working on it, will create a PR very soon

**Comment by OkoliEvans** [comment]
If this becomes available I'll like to get it done @julio4

**Comment by julio4** [comment]
> If this becomes available I'll like to get it done @julio4

Yes it is available, please take the time to read #200. It will be the first section in the AA chapter so this Signature Scheme example will follow it.

**Comment by Jemiiah** [comment]
@julio4 I would love to work on this issue if you allow it 😌🚀

**Comment by julio4** [comment]
> @julio4 I would love to work on this issue if you allow it 😌🚀

@OkoliEvans is already actively working on it currently!
